content/boilerplates/notes/1.2.1.md

```diff
@@ -0,0 +1,16 @@
+## Bug fixes and minor enhancements
+
+- Fix duplicate CRD being generated in the install ([Issue 14976](https://github.com/istio/istio/issues/14976))
+- Allow users to disable istio default retries by setting retries to 0 ([Issue 14900](https://github.com/istio/istio/issues/14900))
+- Improve citadel log ([Pull Request 15032](https://github.com/istio/istio/pull/15032))
+- Add citadel replicaCount flag back and workloadCertTtl to support multi-citadel and short cert TTL for perf test ([Pull Request 15066](https://github.com/istio/istio/pull/15066))
+- Fix Mixer CRD store issue ([Issue 14841](https://github.com/istio/istio/issues/14841))
+- Add HTTP/1.0 support to gateway configuration generation ([Pull Request 15062](https://github.com/istio/istio/pull/15062))
+- Redis filter (this feature is guarded with the feature flag PILOT_ENABLE_REDIS_FILTER, disabled by default):
+    - Enable redis filter ([Pull Request 15049](https://github.com/istio/istio/pull/15049))
+    - Default Redis Cluster ([Pull Request 15052](https://github.com/istio/istio/pull/15052))
+    - Configure Redis latency in microseconds ([Pull Request 15119](https://github.com/istio/istio/pull/15119))
+- Do not overwrite Citadel storage namespace with env var ([Pull Request 15082](https://github.com/istio/istio/pull/15082))
+- Add toleration for Istio components ([Pull Request 15081](https://github.com/istio/istio/pull/15081))
```

Comment:
What does 'toleration' mean for the user?  What can they do with it?

Reply:
need a link to toleration to k8s doc.  I had this in rel note 1.2... Is this a new thing in 1.2.1 too?

Reply:
Looking at the changes, seems adding toleration for certain components in istio.

Reply:
```suggestion
- Add [toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) for Istio components ([Pull Request 15081](https://github.com/istio/istio/pull/15081))
```